tests/src/test/scala/cats/tests/MonoidTests.scala

```diff
@@ -6,8 +6,6 @@ package tests
 class MonoidTests extends CatsSuite {
   {
     Invariant[Monoid]
-    Semigroupal[Monoid]
```

Comment:
why did we lose this? I don't see it?

Reply:
Good catch, I falsely assumed, that because `Monoid` wasn't a valid `InvariantMonoidal`, that it also wasn't a lawful `Semigroupal`. Turns out, it is, and I added it back! Thanks! :)

Reply:
I missed that too. yeah, what broke the law was the `pure` method added in `InvariantMonoidal`